pkgs/applications/misc/wifi4wofi/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, iw
+, networkmanager
+, wofi
+, yad
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wifi4wofi";
+  version = "unstable-2023-08-24";
+
+  src = fetchFromGitHub {
+    owner = "fearlessgeekmedia";
+    repo = "wifi4wofi";
+    rev = "03f809a14fdf10acaeb56b09e3fa2e3bf1fd6200";
+    hash = "sha256-a3nLCYZhXcW9dKdPUBp7z98tM7jaCSk1AaebfIde2tg=";
+  };
+
+  buildInputs = [ iw networkmanager wofi yad ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp $src/${pname} $out/bin/${pname}
```

Comment:
```suggestion
    cp $src/wifi4wofi $out/bin/wifi4wofi
```

Reply:
Please also wrap it to ensure the [dependencies](https://github.com/fearlessgeekmedia/wifi4wofi?tab=readme-ov-file#dependencies) are in PATH